src/main/cpp/events/basesynthevent.cpp

```diff
@@ -264,24 +261,31 @@ void BaseSynthEvent::mixBuffer( AudioBuffer* outputBuffer, int bufferPos,
             calculateBuffers();
     }
 
-    if ( loopStarted && bufferPos >= loopOffset )
+    if ( loopStarted )
     {
```

Comment:
This should never have done anything (looking at 2015 code). Buffer position can never be larger than loopOffset (as loop offset is reached when iterating from the current Sequencer buffer position for the length of the output buffer size). This should ensure that when Sequencer is looping, synthesized events that last for the entire sequence (unlikely) loop correctly.